fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -217,11 +217,7 @@ public PlanFragment visitPhysicalAggregate(
             outputTupleDesc = localAggNode.getAggInfo().getOutputTupleDesc();
         }
 
-        if (aggregate.getAggPhase() == AggPhase.GLOBAL) {
-            for (FunctionCallExpr execAggregateFunction : execAggregateFunctions) {
-                execAggregateFunction.setMergeForNereids(true);
-            }
-        }
```

Comment:
why remove this?

Reply:
already move to ExpressionTranslator.visitAggregateFunction(), there has a line:
```java
boolean isMergeFn = function.isGlobal() && function.isDisassembled();
```